Dockerfile

```diff
@@ -85,6 +85,7 @@ COPY --chown=node:node --from=builder /shieldbattery/server/deployment_files/ent
 # Allow the various scripts to be run (necessary when building on Linux)
 RUN chmod +x ./server/update_server.sh
 RUN chmod +x ./server/testing/run_mailgun.sh
+RUN chmod +x ./server/testing/run_google_cloud.sh
```

Comment:
Can you combine all these `chmod`s into a single line (you should just be able to append the other file paths onto the original command). Docker generates new layers every time you run a command so ideally we want to minimize that